vperfetto-min.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VPERFETTO_EXPORT
+    #ifdef _MSC_VER
+        #define VPERFETTO_EXPORT __declspec(dllexport)
+    #else // _MSC_VER
+        #define VPERFETTO_EXPORT __attribute__((visibility("default")))
+    #endif // !_MSC_VER
+#endif // !VPERFETTO_EXPORT
+
+// Start tracing. This is meant to be triggered when tracing starts in the guest. Use your favorite transport,
```

Comment:
I think I will need a "extern "C"" here so this can be used in C-mostly projects such as Mesa.

Reply:
Done in d0c64ed1e3e6c9db1a03817eba7fb9de3b830aa2